components/TextFields/src/MDCTextInputController.h

```diff
@@ -195,6 +195,16 @@
 /** The text input the controller is affecting. */
 @property(nonatomic, nullable, strong) UIView<MDCTextInput> *textInput;
 
+/**
+ The tintColor applied to the textInput's clear button.
+ See @c UIImageView.tintColor for additional details.
+ */
+@property(nonatomic, null_resettable, strong) UIColor *textInputClearButtonTintColor;
```

Comment:
I think these properties should also be removed from MDCTextInputControllerBase.h so that it's forced to inherit them rather than redeclaring them.

Reply:
Done!